cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -419,13 +455,19 @@ impl xcm_executor::Config for XcmConfig {
 			>,
 		>,
 	);
+	type AssetConverter = (
+		PoolAssetsConverter,
+		TrustBackedSufficientAssetsConverter,
+		ForeignSufficientAssetsConverter,
+	);
 	type ResponseHandler = PolkadotXcm;
 	type AssetTrap = PolkadotXcm;
 	type AssetClaims = PolkadotXcm;
 	type SubscriptionService = PolkadotXcm;
 	type PalletInstancesInfo = AllPalletsWithSystem;
 	type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
 	type AssetLocker = ();
+	// TODO: Might also use `AssetExchanger` instead of `AssetConverter` if API fits.
```

Comment:
Actually now that we're only doing swaps and nothing weird with sufficient assets, I'm looking into this again. I think it would be a win to not have to add one more config item.

Reply:
It does fit! 🎉 Updated PR to use the new config item